app/services/send_transaction_file.service.js

```diff
@@ -0,0 +1,84 @@
+'use strict'
+
+/**
+ * @module SendTransactionFileService
+ */
+
+const Boom = require('@hapi/boom')
+const path = require('path')
+
+const GenerateTransactionFileService = require('./generate_transaction_file.service')
+const SendFileToS3Service = require('./send_file_to_s3.service')
+
+class SendTransactionFileService {
+  /**
+   * Organises the generation and sending of a transaction file:
+   * - Validates that the bill run is in the requisite state;
+   * - Checks that a transaction file is required;
+   * - Calls GenerateTransactionFileService to generate the transaction file;
+   * - Calls SendFileToS3Service to send the transaction file to the S3 bucket;
+   * - Sets the bill run status to 'billed' if everything was successful.
+   *
+   * @param {module:RegimeModel} regime The regime that the bill run belongs to. The regime slug will form part of the
+   * path we upload to.
+   * @param {module:BillRunModel} billRun The bill run we want to send the transaction file for.
+   * @param {function} notify The server.methods.notify method, which we pass in as server.methods isn't accessible
+   * within a service.
+   */
+  static async go (regime, billRun, notify) {
+    this._validate(billRun)
+
+    // If we don't need to generate a file then set the bill status to 'billed' and return early.
```

Comment:
Unfortunately, I think this is incorrect. According to the current code in V1 it should be `billing_not_required`

https://github.com/DEFRA/charging-module-api/blob/main/app/jobs/file_export_job.js#L58-L61

It's not a _great_ naming for the status. But I do think we can't set it to `billed`.